### ISSUE TITLE
Add MPR to list of download sources

### ISSUE THREAD

**Issue by hiddeninthesand** [flacon/flacon.github.io#1]
I maintain a MPR source package for Flacon, and I think Ubuntu/Debian users may find it useful. Package is at https://mpr.makedeb.org/packages/flacon

**Comment by SokoloffA** [comment]
Thank you for your work and your interest in the program!

It would be great if you write a short step-by-step guide on how Ubuntu/Debian users can use your MRP. 

P.S.
The dependencies that are listed on the page are not entirely correct. ffmpeg & hicolor-icon-theme are not required
You can see actual depends on [launchpad](https://launchpadlibrarian.net/583150098/flacon_8.3.0-2impish.dsc)

**Comment by hiddeninthesand** [comment]
Sure! Using the MPR is almost identical to using the AUR on Arch systems, where you use a helper (such as [Tap](https://github.com/hwittenborn/tap)) to feed PKGBUILDs to makedeb, with the only notable difference besides the ecosystem being that you need to install makedeb yourself. There's a great manual for setting up on the makedeb site [here.](https://docs.makedeb.org/mpr/introduction/)

tl;dr: `sudo tap install flacon`

I'll keep the dependencies in mind, I used the dependency list provided by [the AUR source package,](https://aur.archlinux.org/packages/flacon) and according to the comments `ffmpeg` was added because it was required for self-tests on build. I'm not exactly sure why `hicolor-icon-theme` is present, so I may reach out to the AUR maintainer directly for their input.

**Comment by SokoloffA** [comment]
I didn't find the tap program in the [ubuntu repositories](https://packages.ubuntu.com/search?searchon=contents&keywords=tap&mode=exactfilename&suite=impish&arch=any). 
I don't use Debian/Ubuntu, it's hard for me to understand how to install tap/makedeb something else for MRP support.

Could you write as short a step-by-step instruction with commands. Like the following, but for MRP.

> Debian and Linux Mint “Debian” users can use repository at [OpenSUSE Build Service](http://download.opensuse.org/repositories/home:/Sokoloff/).
>
>Open up a terminal and input these commands as root:
>
>**Debian 10**
>```
>echo "deb http://download.opensuse.org/repositories/home:/Sokoloff/Debian_10 ./" > /etc/apt/sources.list.d/flacon.list
>wget http://download.opensuse.org/repositories/home:/Sokoloff/Debian_10/Release.key -O- | apt-key add -
>apt-get update
>apt-get install flacon
>```
